<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="KAGUMPA-G2BH5" name="com.cisco.testscripts.PremiumRouting.LTEnv_Prem_4hrs" tests="1" failures="0" timestamp="19 Oct 2016 20:24:08 GMT" time="567.771" errors="1">
  <testcase name="ltenv_Prem_4hrs" time="567.771" classname="com.cisco.testscripts.PremiumRouting.LTEnv_Prem_4hrs">
    <error type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at lib.cisco.util.ApplicationUtility.prem_SORT_SiteInfo_RMA(ApplicationUtility.java:9508)
	at com.cisco.testscripts.PremiumRouting.LTEnv_Prem_4hrs.ltenv_Prem_4hrs(LTEnv_Prem_4hrs.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at lib.cisco.util.SequentialExecution.runTestNGSequential(SequentialExecution.java:54)
	at lib.cisco.util.Runner.primaryExecution(Runner.java:324)
	at lib.cisco.util.Runner.releaseTesting(Runner.java:290)
	at lib.cisco.util.Runner.main(Runner.java:238)
]]>
    </error>
  </testcase> <!-- ltenv_Prem_4hrs -->
</testsuite> <!-- com.cisco.testscripts.PremiumRouting.LTEnv_Prem_4hrs -->
